bundles/c4c-mock-0.1.0/docs/overview.md

```diff
@@ -0,0 +1,6 @@
+---
+title: SAP C4C Mock
```

Comment:
Shouldn't the title be something like "Overview"

Reply:
no it's the type that your talking about

Reply:
So where the title gets displayed?